taxi/urls.py

```diff
@@ -8,20 +7,20 @@
     DriverDetailView,
     ManufacturerListView,
 )
+from django.contrib.auth import views as auth_views
 
 urlpatterns = [
     path("", index, name="index"),
-    path(
-        "manufacturers/",
-        ManufacturerListView.as_view(),
-        name="manufacturer-list",
-    ),
+    path("manufacturers/", ManufacturerListView.as_view(),
+         name="manufacturer-list"),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
-    path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
-    ),
+    path("drivers/", DriverListView.as_view(),
+         name="driver-list"),
+    path("drivers/<int:pk>/", DriverDetailView.as_view(),
+         name="driver-detail"),
+    path("login/", auth_views.LoginView.as_view(), name="login"),
+    path("logout/", auth_views.LogoutView.as_view(), name="logout"),
 ]
```

Comment:
You don't need to define this urls here. You should use already provided views from django auth.

You imported it in your settings app(main app).